hyperopt/tests/test_spark.py

```diff
@@ -488,8 +489,8 @@ def fn(x):
             self.assertIn(
                 "SparkTrials will block",
                 log_output,
-                """ "SparkTrials will block" missing from log: {log_output}""".format(
-                    log_output=log_output
+                "SparkTrials will block missing from log: log_output".format(
+                    log_output
```

Comment:
that change doesn't do what you intend, I think. It will just print literally "log_output". you need those `{}`.

Reply:
yup, I didn't remove them in purpose 😇 I reverted back to the original version with """, as the string generated had " inside of it and my intention was more about ensuring the max character length rather than changing anything